ibis/expr/operations.py

```diff
@@ -849,7 +849,7 @@ class StringSplit(ValueOp):
 
 
 class StringConcat(ValueOp):
-    arg = Arg(rlz.list_of(rlz.string))
+    arg = Arg(rlz.value_list_of(rlz.string))
```

Comment:
This enforces a `ir.ListExpr`, otherwise I get two pandas errors:

```
node = StringConcat(["ref_0\nPandasTable[table]\n  name: df\n  schema:\n    plain_int64 : int64\n    plain_strings : string\n...egers_strings : map<int64, string>\n    map_of_complex_values : map<string, array<int64>>\n\nScalarParameter[string]"])
kwargs = {'aggcontext': <ibis.backends.pandas.aggcontext.Summarize object at 0x7fd7c868e9a0>, 'clients': [<ibis.backends.pandas...andasClient object at 0x7fd83889aaf0>], 'scope': <ibis.expr.scope.Scope object at 0x7fd828731670>, 'timecontext': None}
    @execute_node.register(ops.Node)
    def execute_node_without_scope(node, **kwargs):
>       raise com.UnboundExpressionError(
            (
                'Node of type {!r} has no data bound to it. '
                'You probably tried to execute an expression without a data '
                'source.'
            ).format(type(node).__name__)
        )
E       ibis.common.exceptions.UnboundExpressionError: Node of type 'StringConcat' has no data bound to it. You probably tried to execute an expression without a data source.
```

I guess this is actually a different issue which we should investigate.

Reply:
https://github.com/ibis-project/ibis/issues/3028